### PR TITLE
Add a General Information label and change the medium link

### PR DESCRIPTION
**Pull request by geektutor**
### Related issues

closes #

### Which environment this relates to
Check your case. In case of any doubts please read about [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html)

- [ ] Test version (https://test.storefrontcloud.io) - this is a new feature or improvement for Vue Storefront. I've created branch from `develop` branch and want to merge it back to `develop`
- [ ] RC version (https://next.storefrontcloud.io) - this is a stabilisation fix for Release Candidate of Vue Storefront. I've created branch from `release` branch and want to merge it back to `release`
- [ ] Stable version (https://demo.storefrontcloud.io) - this is an important fix for current stable version. I've created branch from `hotfix` or `master` branch and want to merge it back to `hotfix`

### Upgrade Notes and Changelog

- [x] No upgrade steps required (100% backward compatibility and no breaking changes)
- [ ] I've updated the [Upgrade notes](https://github.com/DivanteLtd/vue-storefront/blob/develop/doc/Upgrade%20notes.md) and [Changelog](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md) on how to port existing VS sites with this new feature

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and currently important rules acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vue-storefront/blob/master/CONTRIBUTING.md)
- [ ] I read the [TypeScript Action Plan](https://github.com/DivanteLtd/vue-storefront/blob/master/doc/TypeScript%20Action%20Plan.md) and adjusted my PR according to it
- [ ] I read about [Vue Storefront Modules](https://github.com/DivanteLtd/vue-storefront/blob/master/doc/api-modules/about-modules.md) and I am aware that every new feature should be a module

### CODE REVIEW

File: docs/guide/README.md [hunk 1]
```diff
@@ -1,94 +1,94 @@
...
-Some of the backend platforms already have their integrations (Magento 2, Magento 1, CoreShop, BigCommerce, WooCommerce), but you can easily make your own with the [integration boilerplate](https://github.com/DivanteLtd/bigcommerce2vuestorefront
-).
+Some of the backend platforms already have their integrations (Magento 2, Magento 1, CoreShop, BigCommerce, WooCommerce) but you can easily make your own with [integration boilerplate](https://github.com/DivanteLtd/bigcommerce2vuestorefront).
 
 The blue parts on the diagram are responsible for offline cache and will be explained later in the article.
 
-## How does it work?
-
-There are three concepts you need to be familiar with while working with Vue Storefront.
+## How it works?
```
Comment (janmyszkier) [inline docs/guide/README.md]: "How does it work?" was correct english, "How it works?" is not. Please change back.

File: docs/guide/README.md [hunk 2]
```diff
@@ -1,94 +1,94 @@
...
-## How it connect with backend platforms?
-Vue Storefront manages to be platform-agnostic thanks to the [vue-storefront-api](https://github.com/DivanteLtd/vue-storefront-api) and dedicated API connectors for eCommerce backend platforms. The data format in vue-storefront-api is always the same for any platform, which means no matter what eCommerce backend you use, your frontend remains without changes.
+- being platform-agnostic
+- focus on performance
+- mobile-first approach
+- cutting-edge tech
+- no limitations in theming and customization
+- being Open Source with MIT license
+- a lot of focus on developer experience
+- out of the box Server-Side Rendering (for SEO)
```
Comment (janmyszkier) [inline docs/guide/README.md]: "out-of-the-box" (hyphenated) was correct, please change back

File: docs/guide/README.md [hunk 3]
```diff
@@ -1,94 +1,94 @@
...
+- offline mode
 
-It's a great strategy for migrations, since you can easily migrate from one platform to another (or another version, like Magento 1 to 2) without touching your frontend.
+## How it connects with backend platforms?
```
Comment (janmyszkier) [inline docs/guide/README.md]: "How does it connect with backend platforms?" is correct english, please fix.

File: docs/guide/README.md [hunk 4]
```diff
@@ -1,94 +1,94 @@
...
+The mechanism of injecting core business logic into themes is very simple. We are using Vue.js mixins to keep upgradable business logic in the core.
 
-Business logic from the core component can be easily injected into any theme component as a Vue.js mixin. We can inject it into any of our theme components just by importing and adding it as a mixin ( eg `mixins: [Microcart]` is what you need to use core Microcart logic). This is all you need to make use of core business logic inside your theme. With this approach, we can easily ship updates to all core components without breaking your shop.
+Business logic from core component can be easily injected into any theme component as a Vue.js mixin. We can easily inject it into any of our theme components just by importing it and adding it as a mixin ( eg `mixins: [Microcart]` is what you need to use core Microcart logic). This is all you need to make use of core business logic inside your theme. With this approach, we can easily ship updates to all core components without breaking your shop.
```
Comment (janmyszkier) [inline docs/guide/README.md]: why was "the" removed from "the core component"? It was correct English. Please change back.

File: docs/guide/README.md [hunk 5]
```diff
@@ -1,94 +1,94 @@
...
 
-The easiest way to create your own theme is to create a copy of the default one, change its name in its `package.json` file, change the active theme in `config/local.jso`n and run `yarn` to make lerna linking (which we are using for monorepos).
+The easiest way to create your own theme is to create a copy of the default one, change it's name in it's `package.json` file, change active theme in `config/local.json` and run `yarn` to make lerna linking (which we are using for monorepos).
```
Comment (janmyszkier) [inline docs/guide/README.md]: wrong replacement of "its" with "it's" please change back. Should be: 
change its name in it's `package.json` file

File: docs/guide/README.md [hunk 6]
```diff
@@ -1,94 +1,94 @@
...
 
-We managed to do this by making heavy use of the browser cache. 
+Vue Storefront is still working even while a user is offline.
+
+We managed to do this by making heavy use of the browser cache.
 For the static assets (only prod) we are using the sw-precache plugin (config can be found in `core/build/webpack.prod.sw.config.js` ). They are cached in Service Worker and can be inspected under `Application/Cache Storage` tab of your Developer Tools.
 
 :::warning
-Please keep in mind that Service Worker is working only on production mode.
+Please mind that Service Worker is working only on production mode.
```
Comment (janmyszkier) [inline docs/guide/README.md]: why was "keep in" from "keep in mind" removed? It made perfect sense, please change back.

File: docs/guide/README.md (hunk 2 above)
Comment (geektutor) [reply docs/guide/README.md]: Got it

**Pull request closed without merging**